nipype/utils/filemanip.py

```diff
@@ -69,6 +71,24 @@ def _patch_resolve(self, strict=False):
 
     Path.old_resolve = Path.resolve
     Path.resolve = _patch_resolve
+
+    if not hasattr(Path, 'write_text'):
+        def _write_text(self, text):
+            with open(str(self), 'w') as f:
+                f.write(text)
+        Path.write_text = _write_text
+
+    try:
+        (Path(gettempdir()) / 'exist_ok_test').mkdir(exist_ok=True)
```

Comment:
This is just going to pollute the temporary directory on every nipype import. Might as well clean up after ourselves:

```suggestion
        with tempfile.TemporaryDirectory() as tmpdir:
            (Path(tmpdir) / 'exist_ok_test').mkdir(exist_ok=True)
```